test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/services/chat-states/ChatStates.spec.js

```diff
@@ -117,6 +134,30 @@ describe("Chat States", () => {
             expect(playMessageRecievedSound).toBeCalledTimes(1);
         });
 
+        it("does not play sound on incoming agent message when user has sound turned off", () => {
```

Comment:
this test could be the 'if there is no soundElement on the page to drop in the else block after the null check, and there should be another test to add the expected elements to the document with the 'inactive'  attribute in the same pattern as the "plays sound on incoming agent message when user has sound turned on" test